README.md

```diff
@@ -21,29 +21,40 @@ _Xamarin Forms provides cross-platform APIs, but this project are only worked on
 - CirclePage : A subclass of Xamarin.Forms.Page that can show Circular ProgressBar and Circular Slider and the button on the bottom of the screen. it can also show MenuItems on the circular menu.(MoreOption)
 - CircleProgressBarSurfaceItem : A control fits in the circular screen. it can be shown in CirclePage only.
-- CircleSliderSurfaceItem : A Slider control that responds to the bezel action and fits in the circular screen.
 - CircleScrollView : A subclass of Xamarin.Forms.ScrollView that can be scrolled by the bezel action.
+- CircleSliderSurfaceItem : A slider control that responds to the bezel action and fits in the circular screen.
 - CircleStackLayout : A container to layout children linear in the circular area.
 - CircleStepper : A control to select a number of steps fits in the circular screen.
 - CircleSurfaceEffectBehavior : The CircleSurfaceEffectBehavior is an effect which allows you to insert views that require CircleSurface.
-- ContextPopupEffectBehavior : the behavior to show a small popup has one or two button sticky with any control.
+- CircleSurfaceItem : A control to represent the items in the CircleSurface.
+- CircleSurfaceView : A subclass of Xamarin.Forms.View that can have several CircleSurfaceItem as a View not Page.
+- CircleToolbarItem : A subclass of Xamarin.Forms.ToolbarItem that can show toolbar item for fits in the circular screen.
+- CircularShell : A subclass of Xamarin.Forms.Shell that have additional properties for circular screen.
+- ColorMenuItem : A subclass of Xamarin.Forms.MenuItem that controls the background color of the button used in the popup control.
+- ContentButton : A button that allows you to customize the view to be displayed.
+- ContextPopupEffectBehavior : The behavior to show a small popup has one or two button sticky with any control.
+- FlatViewCell : A subclass of Xamarin.Forms.ViewCell without fish-eye effect.
+- GoogleMapView : A GoogleMapView is used to display google javascript map.
 - IndexPage : A subclass of multiple pages that can slide child page horizontal and has dots on top of the screen for the number of child pages.
 - InformationPopup : A popup has a control to show progress and one button on the bottom side of the circular screen.
-- IRotationReceiver : A event receiver for using the bezel action.
-- Radio : A radio control.
-- GoogleMapView : A GoogleMapView is used to display google javascript map.
+- MediaPlayer : A class that provides the essential components to play the media contents.
 - MediaView : A MediaView is used to display a video on the screen.
 - PopupEntry : The PopupEntry is a class that extends Xamarin.Forms.Entry. It makes a new layer when editing text on the entry.
-- Toast : A popup for simple feedback.
-- TwoButtonPopup : A popup has two buttons on the left and right side of the circular screen.
+- Radio : A radio control that displays one or more options, and allows the user choose only on of them.
```

Comment:
remove `,`.
allows the user choose -> allows the user to choose
on -> one

Reply:
Radio is moved to new paragraph for "deprecated APIs", and description is removed.